src/main/java/seedu/type/IncomeList.java

```diff
@@ -37,6 +46,7 @@ public void updateRecurrence() {
 
     @Override
     public boolean add(Income income) {
```

Comment:
now that i realise can this method name not sound like boolean?

Reply:
I think should be fine since we are overriding `ArrayList` class, so it must be of the same method signature